assets/scss/stripe.scss

```diff
@@ -1,5 +1,123 @@
-@include responsiveBase() {
-	.stripe{
+/*doc
+---
+title: Stripe
+---
+A `.stripe` class is rendered by the `<Stripe>` component to
+visually divide the page with borders or stripes of color.
+
+To align content within a stripe, nest children in a
+`<Bounds>` component.
+
+#### Stripe variants
+A class of `.stripe` is default; the following classes are
+optional variants:
```

Comment:
out of curiosity, how does this relate to what is already being imported in SQ2? Do we have some redundancy?